### ISSUE TITLE
Issue in reconstructing amcharts in React

### ISSUE THREAD

**Issue by sipubohre** [amcharts/amcharts3-react#81]
We are constructing multiple charts in a layout having option to remove chart from that layout. We constructed two charts one is Serial type and other one is gauge type. When we remove first chart that is serial type. then an error Is showing from Amcharts.js
"Cannot read property 'length' of undefined at b.processGraphs (amcharts.js:9)"
this is code how we using it.

![capture1](https://user-images.githubusercontent.com/37571091/45737848-844bf500-bc0c-11e8-9ca7-c024681ec5cb.PNG)
 React element is generated successfully but when it goes in render it crashes.

**Comment by amcharts** [comment]
> New comment from Zendesk by **Martynas Majeris** on ticket **39491**.
> (replying here will automatically notify amCharts support agent)

Request #39492 "Issue while reconstructing chart..." was closed and merged into this request. Last comment in request #39492:

Hi Team,


                We are facing an issue while reconstructing charts in Reactjs. when we portrait serial and gauge chart using the library then all charts are working fine. however when we remove one of the chart from layout then an error is showing "length of undefined" in amcharts.js.


We are constructing multiple charts in a layout having option to remove chart from that layout. We constructed two charts one is Serial type and other one is gauge type. When we remove first chart that is serial type. then an error Is showing from Amcharts.js
"Cannot read property 'length' of undefined at b.processGraphs (amcharts.js:9)"
this is code how we using it.

**Comment by amcharts** [comment]
> New comment from Zendesk by **Devendra Bohre** on ticket **39491**.
> (replying here will automatically notify amCharts support agent)

yes, I have used amcharts 4 earlier, but that was not fully functional in my project, so I am using amcharts3 currently. Is there any issue with this version?
